internal/graph/checkingresourcestream.go

```diff
@@ -182,6 +183,10 @@ type checkingResourceStream struct {
 	// and waitForPublishing() (after reachable resources has completed).
 	lastResourceCursor *v1.Cursor
 
+	// dispatchesToBeReported is the number of dispatches that were skipped from being reported due
+	// to a resource being filtered, and whose count has to be attached to the next outgoing result.
+	dispatchesToBeReported atomic.Uint32
+
```

Comment:
please track cached dispatches as well until we figure out the long term solution for them in this kind of stochastic reporting case

Reply:
Added